File: RMS-BackOfficePortal/src/app/core/models/otc-bal-info.ts
export interface OTCBalInfo{
    branch_cd: string;
    bal_type: string;
    no_of_counters: number | 0;
    no_of_txn: number | 0;
    total: number | 0;
    total_emv: number | 0;
    total_phy: number | 0;
    total_cash: number | 0;
    total_che: number | 0;
    total_bd: number | 0;
    total_mo: number | 0;
    no_of_rcpt_can: number | 0;
    bal_date: Date;

    status: string | 'N/A';
}

File: RMS/rms-service/src/main/java/com/maven/rms/models/OTCBalInfo.java
package com.maven.rms.models;

import java.math.BigDecimal;
import java.sql.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OTCBalInfo {
    
    private String branch_cd;
    private Date dt_bal;
    private String bal_type;
    private Integer no_of_counters;
    private Integer no_of_txn;
    private BigDecimal total;
    private BigDecimal total_emv;
    private BigDecimal total_phy;
    private BigDecimal total_cash;
    private BigDecimal total_che;
    private BigDecimal total_bd;
    private BigDecimal total_mo;
    private Integer no_of_rcpt_can;

    private String status;

}
